examples/jest_integration/package.json

```diff
@@ -2,6 +2,7 @@
 	"name": "jazzerjs-jest-integration-example",
 	"version": "1.0.0",
 	"description": "An example showing how Jazzer.js integrates with Jest",
+	"type": "module",
```

Comment:
If I understand correctly, this turns this example to a ES module project. Should we also include a Common JS example, as ES module support is still experimental in Jest?

Reply:
Good catch! The ESM project is `protobufjs`. Removed `type` from `jest_integration`.